src/param/param_slash.c

```diff
@@ -256,23 +340,96 @@ static int cmd_set(struct slash *slash) {
         optparse_del(parser);
 		return SLASH_EINVAL;
 	}
+	
+	// Create a queue, so that we can set the param in a single packet.
+	param_queue_t queue;
+	char queue_buf[PARAM_SERVER_MTU];
+	param_queue_init(&queue, queue_buf, PARAM_SERVER_MTU, 0, PARAM_QUEUE_TYPE_SET, 2);
```

Comment:
Could this fail, and render the `queue` invalid?

Reply:
No, this cannot fail. By also looking at other implementations of queue, this seems to be the way to go.